### PR TITLE
Release notes for BK 4.15

### PR DESCRIPTION
**Pull request by dlg99**
Descriptions of the changes in this PR:
### Motivation

release notes for BK release 4.15

### Changes

Doc updates per release procedure.

### CODE REVIEW

**Review by zymap — APPROVED** [review]

File: site/docs/4.15.0/overview/releaseNotes.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
+---
+title: Apache BookKeeper 4.15.0 Release Notes
+---
+
+Release 4.15 addresses multiple CVEs, adds more configuration options, extends REST API,
```
Comment (eolivelli) [inline site/docs/4.15.0/overview/releaseNotes.md:5]: I would say:
"Release 4.15 addresses multiple CVEs" -> "includes many upgrades to third party libraries marked with CVEs"

**Review by eolivelli — APPROVED** [review]
LGTM
Great work 
I left just one minor comment

File: site/docs/4.15.0/overview/releaseNotes.md (hunk 1 above)
Comment (dlg99) [reply site/docs/4.15.0/overview/releaseNotes.md]: fixed

**Comment by dlg99** [comment]
@nicoloboschi i'll pass on site3/ updates for now, we can do it as a separate PR.

**Review by nicoloboschi — APPROVED** [review]
LGTM 
we can follow up with site3 in another pull

**Comment by dlg99** [comment]
site3/docusarus worked after @nicoloboschi 's help

```
export LATEST_RELEASED=4.14.4
export NEW_RELEASE=4.15.0

brew install yarn
cd ~/src/bookkeeper/site3/website
yarn install
yarn run docusaurus docs:version $NEW_RELEASE
```
then edit src/pages/release-notes.md

**Review by hangc0276 — APPROVED** [review]

**Pull request merged**